### DISCUSSION TITLE
Problem installer Whisper offline

### DISCUSSION THREAD

**Discussion by cayerpm** [openai/whisper#1556] [Q&A]
I need to install Whisper on offline Windows computers. 

I am able to install all dependancies, but when I try to install the whisper package I get an error stating that the installer could not find a valid version of setuptools:

`
ERROR: Could not find a version that satisfies the requirement setuptools>=40.8.0 (from versions: none)
ERROR: No matching distribution found for setuptools>=40.8.0
`
However, setuptools is indeed installed with version 67.6.1:

`D:\Whisper\v20230314\whisper-dependances>pip list
Package            Version
------------------ -----------
blobfile           2.0.2
certifi            2022.12.7
charset-normalizer 3.1.0
colorama           0.4.6
ffmpeg-python      0.2.0
filelock           3.12.0
future             0.18.3
idna               3.4
Jinja2             3.1.2
llvmlite           0.39.1
lxml               4.9.2
MarkupSafe         2.1.2
more-itertools     9.1.0
mpmath             1.3.0
networkx           3.1	
numba              0.56.4
numpy              1.23.5
pip                22.3.1
pycryptodomex      3.17
regex              2023.3.23
requests           2.28.2
**setuptools         67.6.1**
sympy              1.11.1
tiktoken           0.3.1
torch              2.0.0+cu118
tqdm               4.65.0
typing_extensions  4.5.0
urllib3            1.26.15
`

I tried uninstalling everything, including Python, and restarting from scrtch and I always get the same issue.

Anyone have an idea?

Thx

PM

**Comment by phineas-pta** [comment]
#1463

**Reply by cayerpm** [reply]
Yeah, I read that post, but this does not solve my issue.
